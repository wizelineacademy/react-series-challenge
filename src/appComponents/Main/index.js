import React from 'react';
import { Provider } from 'react-redux';
import store from '../../utils/store';
import Routes from '../Routes';
import Layout from '../../components/Layout';
import StyledApp from './styles.js';

const App = () => (
  <Provider store={store}>
    <StyledApp className="App">
      <Layout>
        <Routes />
      </Layout>
    </StyledApp>
  </Provider>
);

export default App;
