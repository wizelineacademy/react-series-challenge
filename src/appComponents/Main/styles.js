import styled from 'styled-components';

const StyledApp = styled.div`
  text-align: center;
`;

export default StyledApp;
