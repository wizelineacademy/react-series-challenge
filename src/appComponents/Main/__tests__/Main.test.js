import React from 'react';
import ReactDOM from 'react-dom';
import Main from '../../Main';

describe('<Main />', () => {
  it('App renders without crashing', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
