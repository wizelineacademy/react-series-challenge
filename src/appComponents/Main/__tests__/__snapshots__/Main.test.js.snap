// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> App renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="App"
      >
        <Layout>
          <Routes />
        </Layout>
      </div>,
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Layout>
          <Routes />
        </Layout>,
        "className": "App",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Routes />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="App"
        >
          <Layout>
            <Routes />
          </Layout>
        </div>,
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Layout>
            <Routes />
          </Layout>,
          "className": "App",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Routes />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
